pkg/resources/model/system.go

```diff
@@ -164,10 +175,24 @@ func GetFlowMatchFromSpec(namespace string, matches interface{}) ([]types.FlowMa
 	return flowMatches, nil
 }
 
-func FlowDispatcher(flowCr interface{}) (*CommonFlow, error) {
+func (l *LoggingResources) FlowDispatcher(flowCr interface{}) (*CommonFlow, error) {
 	var commonFlow *CommonFlow
 	var err error
 	switch f := flowCr.(type) {
+	case *v1beta1.DefaultFlowSpec:
+		id := fmt.Sprintf("logging:%s:%s", l.logging.Namespace, l.logging.Name)
+		flow, err := types.NewFlow([]types.FlowMatch{}, id, l.logging.Name, l.logging.Namespace)
```

Comment:
I'm not sure this would work without an actual flow match defined. I would need to create a test case for this, but without digging deeper, I think we just need an empty FlowMatch item put in the list.

Reply:
nevermind, I didn't think this through carefully